### ISSUE TITLE
Unable to sync gradle after updating to 5.15.0

### ISSUE THREAD

**Issue by jvanderwee** [realm/realm-java#6610]
#### Goal

Update to realm-java 5.15.0.

#### Actual Results
sync failes
```
Caused by: groovy.lang.MissingPropertyException: Could not get unknown property 'url' for object of type org.gradle.api.internal.artifacts.repositories.DefaultFlatDirArtifactRepository.
	at org.gradle.internal.metaobject.AbstractDynamicObject.getMissingProperty(AbstractDynamicObject.java:87)
	at org.gradle.internal.metaobject.AbstractDynamicObject.getProperty(AbstractDynamicObject.java:61)
	at org.gradle.api.internal.artifacts.repositories.DefaultFlatDirArtifactRepository_Decorated.getProperty(Unknown Source)
	at org.codehaus.groovy.runtime.callsite.PogoGetPropertySite.getProperty(PogoGetPropertySite.java:49)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callGetProperty(AbstractCallSite.java:298)
	at io.realm.gradle.Realm$_apply_closure1$_closure2.doCall(Realm.groovy:79)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.codehaus.groovy.reflection.CachedMethod.invoke(CachedMethod.java:104)
	at groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:326)
	at org.codehaus.groovy.runtime.metaclass.ClosureMetaClass.invokeMethod(ClosureMetaClass.java:264)
	at groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1041)
	at groovy.lang.Closure.call(Closure.java:411)
	at groovy.lang.Closure.call(Closure.java:427)
	at org.gradle.api.specs.internal.ClosureSpec.isSatisfiedBy(ClosureSpec.java:33)
	at org.gradle.api.internal.collections.CollectionFilter.filter(CollectionFilter.java:46)
	at org.gradle.api.internal.collections.FilteredCollection$FilteringIterator.findNext(FilteredCollection.java:122)
	at org.gradle.api.internal.collections.FilteredCollection$FilteringIterator.next(FilteredCollection.java:140)
	at org.gradle.api.internal.DefaultDomainObjectCollection.findAll(DefaultDomainObjectCollection.java:442)
	at org.gradle.api.internal.DefaultNamedDomainObjectList.findAll(DefaultNamedDomainObjectList.java:160)
	at org.gradle.api.NamedDomainObjectList$findAll.call(Unknown Source)
	at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCall(CallSiteArray.java:47)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:115)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:127)
	at io.realm.gradle.Realm$_apply_closure1.doCall(Realm.groovy:78)
```

#### Steps & Code to Reproduce
1. Add `classpath 'io.realm:realm-gradle-plugin:5.15.0'` to dependencies
2. Sync

#### Version of Realm and tooling

Realm version(s): 5.15.0

Realm Sync feature enabled: No

Android Studio version: 3.5

Android Build Tools version: 3.5.0

Gradle version: 5.4.1

Which Android version and device(s): N/A

**Comment by cmelchior** [comment]
Hmm, can you post your `build.gradle` file?

**Comment by ivabra** [comment]
@cmelchior It happens when you have a `flatDir` repository in your project:
```
repositories {
 
  flatDir {
      dirs 'libs'
  }
  google()

}
```
Tooling is the same.

**Comment by ayltai** [comment]
This line tries to get repository `url` but there is no `url` property when using `flatDir`:
https://github.com/realm/realm-java/commit/00698d17d1348160e002af961f4f5722f7faaf26#diff-fbc37097ca151e18a6db623e59ebdba8R79

**Comment by cmelchior** [comment]
Ups...The curse of dynamic languages. I'll create a fix and release 5.15.1 today.

**Comment by cmelchior** [comment]
5.15.1 has been released with a fix for this
